app/controllers/benefits/accommodation/NonQualifyingRelocationBenefitsAmountController.scala

```diff
@@ -54,12 +54,9 @@ class NonQualifyingRelocationBenefitsAmountController @Inject()(authAction: Auth
         redirectBasedOnCurrentAnswers(taxYear, employmentId, optCya,
           EmploymentBenefitsType)(nonQualifyingRelocationBenefitsAmountRedirects(_, taxYear, employmentId)) { cya =>
           val cyaAmount = cya.employment.employmentBenefits.flatMap(_.accommodationRelocationModel.flatMap(_.nonQualifyingRelocationExpenses))
+          val form = fillForm(formsProvider.nonQualifyingRelocationAmountForm(request.user.isAgent), cyaAmount)
 
-          val form = fillFormFromPriorAndCYA(formsProvider.nonQualifyingRelocationAmountForm(request.user.isAgent), prior, cyaAmount, employmentId)(
-            employment => employment.employmentBenefits.flatMap(_.benefits.flatMap(_.nonQualifyingRelocationExpenses))
-          )
-
-          Future.successful(Ok(nonQualifyingRelocationBenefitsView(taxYear, form, cyaAmount, employmentId)))
+          Future.successful(Ok(pageView(taxYear, form, cyaAmount, employmentId)))
```

Comment:
It seems the same thing about removing the `amount` applies here, and all the rest of the controllers and views.

Reply:
I will change them all